lib/screens/v2/widget_instance/departures.ex

```diff
@@ -100,18 +107,24 @@ defmodule Screens.V2.WidgetInstance.Departures do
 
     text =
       if is_only_section do
+        time_range =
+          if headsign == "Ashmont/Braintree" do
```

Comment:
Is it worth making this conditional on the length of the headsign instead of hardcoding it for Ashmont/Braintree?

Reply:
Hmm, I'm not sure. A part of me thinks we want this to be case by case. I think design didn't want to abbreviate this headsign so we went this direction, but we may want to treat future headsigns differently. Another hesitation is the unreliability of looking at length when not using a monospace font. I will check with design though so I'm not making any incorrect assumptions.

Reply:
Got an answer back from Paul. The one-off is enough for now since it's the only headsign we anticipate needing to hardcode for.

Reply:
Sounds good!